.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "cSpell.words": [
+        "shotform"
```

Comment:
I thought cSpell was "smart" enough to figure out that ShotForm is two valid words when camel or pascal case? Are you seeing issues with this somewhere?

Reply:
Is this from my spellcheck?  If so, it could have been when I made a scss file. I forgot to pascal case it. I deleted the file, because we do not use it.